cli/src/cluster_query.rs

```diff
@@ -742,6 +742,8 @@ pub fn process_catchup(
         }
     };
 
+    let first_node_slot = get_slot_while_retrying(&node_client)?;
+    let mut interval_count = 1;
```

Comment:
let's initialize this as 0 so, that we can test the empty sting case easily. Also this should really be 0 because we haven't started. :)